python/sglang/srt/model_executor/model_runner.py

```diff
@@ -590,6 +596,7 @@ def init_memory_pool(
             max_context_len=self.model_config.context_len + 4,
             device=self.device,
             use_records=False,
+            enable_memory_saver=self.server_args.enable_memory_saver,
```

Comment:
See this comment https://github.com/sgl-project/sglang/pull/2630#discussion_r1912734551

Reply:
Oops I misunderstood your question. I am worried maybe not, because it can happen that we do have some tensors inside req pool that needs to be preserved across memory release. (The torch_memory_saver does not offload things to cpu, instead it just throws away the content)